library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -485,7 +485,7 @@ void PlatformBridgeFilter::FilterBase::onResume() {
     pending_trailers = &bridged_trailers;
   }
 
-  ENVOY_LOG(trace, "PlatformBridgeFilter({})->on_resume_*", parent_.filter_name_);
+  ENVOY_LOG(debug, "PlatformBridgeFilter({})->on_resume_*", parent_.filter_name_);
```

Comment:
We can probably leave this at trace right? I don't think we'll learn much from it in this context.